src/main/java/com/teamabnormals/abnormals_core/core/api/IAgeableEntity.java

```diff
@@ -1,15 +1,29 @@
 package com.teamabnormals.abnormals_core.core.api;
 
+import net.minecraft.entity.Entity;
+
 /***
- * @author tessdotcpp
- * Use to make an entity that doesn't extend AgeableEntity compatible with Quark's potato poisoning.
+ * @author abigailfails
+ * Use to make an entity that doesn't extend AgeableEntity compatible with Quark's potato poisoning and
+ * Savage & Ravage's Growth & Youth Potions.
  */
 public interface IAgeableEntity {
+
     /**
-     * Sets the growing age of the entity. With a negative value it's considered a child; use this method to check for
-     * an age of 0 or greater and trigger the necessary changes.
-     */
-    void setGrowingAge(int age);
+     * If the entity can grow to a higher stage, resets any progress made towards it (e.g. age timer).
+     * */
+    void resetGrowthProgress();
 
-    int getGrowingAge();
+    /**
+     * @return whether the entity has another stage it can grow or regress to depending on isGrowingUp.
+     * @param isGrowingUp true checks for a higher growth stage, false checks for a lower one.
+     * */
+    boolean canAge(boolean isGrowingUp);
```

Comment:
The formatting for the docs of this method are incorrect and applies to the other methods in this interface.
```suggestion
/**
 * @param isGrowingUp true checks for a higher growth stage, false checks for a lower one.
 * @return whether the entity has another stage it can grow or regress to depending on isGrowingUp.
 */
 boolean canAge(boolean isGrowingUp);
```